source/evp_fetch.c

```diff
@@ -68,7 +68,7 @@
 size_t *counts;
 OSSL_TIME max_time;
 
-int err = 0;
+int error = 0;
```

Comment:
I'm still at a loss for why this had to be done everywhere.  I just reverted the changes to evp_fetch.c regarding this conversion, and with Visual Studio 17 2022, the perftools repo built fine.  What compiler version produced errors for you?

Reply:
If I rememer correctly the change was prompted by thread affinity which is gone. so perhaps we can drop this changeset from this PR too.